ci/terraform/oidc/api-gateway.tf

```diff
@@ -611,7 +611,7 @@ resource "aws_api_gateway_method" "orch_frontend_proxy_method" {
 
 data "aws_cloudformation_stack" "orch_frontend_stack" {
   count = var.orch_frontend_api_gateway_integration_enabled ? 1 : 0
-  name  = "${var.environment}-orch-frontend"
+  name  = replace("${var.environment}-orch-f-deploy", "authdev1", "dev")
```

Comment:
is there a reason you've changed `frontend` to `f-deploy`? I'm wondering if this might mess with the sandpit deployment

Reply:
@isaac-GDS changed the stack name in the pipeline, not sure of the reasoning?

Reply:
Since updated stack names to be more descriptive - not able to have `{env}-orch-frontend-deploy` as the stack name has a character limit.